plugins/techdocs/src/reader/transformers/scrollIntoAnchor.ts

```diff
@@ -15,17 +15,24 @@
  */
 
 import type { Transformer } from './transformer';
+import { SHADOW_DOM_STYLE_LOAD_EVENT } from '@backstage/plugin-techdocs-react';
 
 export const scrollIntoAnchor = (): Transformer => {
   return dom => {
-    setTimeout(() => {
-      // Scroll to the desired anchor on initial navigation
-      if (window.location.hash) {
-        const hash = window.location.hash.slice(1);
-        // fix invalid selector error for anchor starting with number
-        dom?.querySelector(`[id="${hash}"]`)?.scrollIntoView();
-      }
-    }, 200);
+    dom.addEventListener(
+      SHADOW_DOM_STYLE_LOAD_EVENT,
+      function handleShadowDomStyleLoad() {
+        if (window.location.hash) {
+          const hash = window.location.hash.slice(1);
+          // fix invalid selector error for anchor starting with number
```

Comment:
I know this comment is old, but I don't really understand it, do you? Like Is it a todo or is it describing the row beneath? Maybe we can delete it or clarify it if possible.

Reply:
we can delete this comment IMO, it refers to implementation details for a fix in the previous commit that changed the queryselector from: ``dom?.querySelector(`#${hash}`)``  to ``dom?.querySelector(`[id="${hash}"]`)``in commit f90ffdc8